app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -111,6 +124,17 @@ def process_server_entity(server, entity)
         end
       end
 
+      def process_availability(availability)
+        case
+        when availability.nil?, availability['value'] == 'unknown'
```

Comment:
It *should* always arrive downcase[1]. But then, is only one call and might be worth doing that.

I read somewhere that "Running .empty? on something that is nil will throw a NoMethodError", I could test both if that's the case. "when availability.nil?, availability.empty?,..."

[1] https://github.com/hawkular/hawkular-metrics/blob/master/core/metrics-model/src/main/java/org/hawkular/metrics/model/AvailabilityType.java#L28-L34